pkg/controller/tas/topology_ungater.go

```diff
@@ -93,7 +94,7 @@ func (r *topologyUngater) setupWithManager(mgr ctrl.Manager, cfg *configapi.Conf
 		expectationsStore: r.expectationsStore,
 	}
 	return TASTopologyUngater, ctrl.NewControllerManagedBy(mgr).
-		Named(TASTopologyUngater).
+		Named(strings.ReplaceAll(TASTopologyUngater, "-", "_")).
```

Comment:
Could we just adjust the names rather that do the replacements here?

Reply:
SGTM